Core/Core/Kits/KitManager.cs

```diff
@@ -22,10 +22,10 @@ public static string KitsFolder
     {
       get
       {
-        if (_kitsFolder != null)
-          return _kitsFolder;
+        if (_kitsFolder == null)
+          _kitsFolder = Path.Combine(Helpers.InstallSpeckleFolderPath, "Kits");
 
-        return Path.Combine(Helpers.InstallSpeckleFolderPath, "Kits");
+        return _kitsFolder;
```

Comment:
This is not really required, but it seemed the lazy loading wasn't done correctly.